src/test/java/LoginTests.java

```diff
@@ -15,9 +15,15 @@ public static void LoginEmptyEmailPasswordTest() {
         WebDriver driver = new ChromeDriver();
         driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
 
-        String url = "https://testpro.io/";
+        String url = "https://bbb.testpro.io/";
         driver.get(url);
+
+
+
+
         Assert.assertEquals(driver.getCurrentUrl(), url);
         driver.quit();
+
+
```

Comment:
Please, add the class HomeWork16  in TestNG.xml

For example

```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
<suite name="TestNG">
    <test name="Example Test" preserve-order="false">
        <classes>
            <class name="HomeWork16"/>
        </classes>
    </test>
</suite>
```

![image](https://user-images.githubusercontent.com/104761482/221347867-833bb2ff-c1cd-4d33-8f1a-0c1e98db26c8.png)